variants/backend-base/example.env.tt

```diff
@@ -26,3 +20,8 @@ PORT=3000
 # SENTRY_DSN=http://public@example.com/project-id
 SENTRY_CSP_HEADER_REPORT_ENDPOINT=https://SOMECODE.ingest.sentry.io/api/SOMENUMS/security/?sentry_key=SOMETHING
 SENTRY_ENV=development
+
+# run `bin/rails db:encryption:init` to create real versions of these secrets
+ACTIVE_RECORD_ENCRYPTION_PRIMARY_KEY="TODO_CHANGE_ME"
```

Comment:
~switching to using `ENV.fetch` would mean that these error if not present - I'm wondering if we should actually comment these out to trigger that? (we could also generate them as part of creating the template right like we do `RAILS_SECRET_KEY_BASE)~

actually, we should generate these like we do with `RAILS_SECRET_KEY_BASE` for the same reason: otherwise dev envs would be encrypted with different values which'd make it harder to pair/share/etc.